### PR TITLE
Switch UAA over to PXC for database management.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref https://jira.suse.com/browse/CAP-260

Controlling SCF PR https://github.com/SUSE/scf/pull/2465

### CODE REVIEW

File: container-host-files/etc/scf/config/role-manifest.yml [hunk 1]
```diff
@@ -233,7 +321,7 @@ configuration:
     ip: '"((IP_ADDRESS))"'
     networks.default.dns_record_name: '"((DNS_RECORD_NAME))"'
     networks.default.ip: '"((IP_ADDRESS))"'
-    properties.cf_mysql.mysql.advertise_host: mysql-((KUBE_COMPONENT_INDEX)).mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+    #properties.cf_mysql.mysql.advertise_host: mysql-((KUBE_COMPONENT_INDEX)).mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
```
Comment (f0rmiga) [inline container-host-files/etc/scf/config/role-manifest.yml]: Why do we have this line commented out?

**Review by f0rmiga — CHANGES_REQUESTED** [review]

File: container-host-files/etc/scf/config/role-manifest.yml (hunk 1 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml]: I should remove it. It does not seem to be required. 
And now I don't remember how it was handled by the main mysql/pxc bump. 
Was this present in the original main mysql ? And removed in the bump ?

Comment (f0rmiga) [reply container-host-files/etc/scf/config/role-manifest.yml]: Indeed, I removed it from the original move to PXC. https://github.com/SUSE/scf/pull/2390/files#diff-a65976ef35c13467fe140d46c6ca0d3aL2777

Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml]: Ok, will remove completely also.

Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml]: Done in commit c7c018469b9310db9fa6bce378b854347b435e9f

**Review by f0rmiga — APPROVED** [review]

**Pull request merged**